CHANGELOG.md

```diff
@@ -17,9 +17,15 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
     - `sdk.alerts.resolve()`
     - `sdk.alerts.reopen()`
 
+## Fixed
+
+- Issue that in rare circumstance could cause py42 to exhaust network sockets. This could sometimes occur if you were running a multi-threaded program that communicated with many Code42 storage nodes.
+
 ### Added
 
-- Debug logs for restore progress during the method call `py42.archive.stream_from_back()`.
```

Comment:
I feel like this might be a good change to make users aware of, if they want to track progress. Else, how are they supposed to know?

Reply:
it isn't removed, the method name was misspelled (should be `backup()`), so I corrected that.

Reply:
Oh I see it is below